orchagent/aclorch.cpp

```diff
@@ -153,6 +153,244 @@ static const acl_capabilities_t defaultAclActionsSupported =
     }
 };
 
+static acl_table_action_list_lookup_t defaultAclActionList = 
+{
+    {
+        // L3
+        TABLE_TYPE_L3,
+        {
+            {
+                ACL_STAGE_INGRESS,
+                AclActionCapabilities
+                {
+                    {
+                        SAI_ACL_ACTION_TYPE_PACKET_ACTION
```

Comment:
There's a redirect action used for L3, L3V6 tables

Reply:
Fixed. I added `SAI_ACL_ACTION_TYPE_REDIRECT` for `L3` and `L3V6`. Thanks